### PR TITLE
add new device response for indicate next cal time

### PR DESCRIPTION
**Pull request by jannast**
Rework device manipulation IndicateTimeOfNextCalibrationToUser with new response IndicateTimeOfNextCalibrationToUserResponse.

# Checklist

The following aspects have been respected by the author of this pull request, confirmed by both pull request assignee **and** reviewer:

* Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* README update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

File: src/t2iapi/device/types.proto [hunk 1]
```diff
@@ -79,3 +79,12 @@ enum CalibrationState{
     CALIBRATION_STATE_CALIBRATED = 3;
     CALIBRATION_STATE_OTHER = 4;
 }
+
+/*
+Represents a container for MDIB version attributes.
+ */
+message MdibVersion {
+    string sequence_id = 1;     // SequenceId when finishing the rpc
+    uint64 instance_id = 2;     // InstanceId when finishing the rpc
```
Comment (ldeichmann) [inline src/t2iapi/device/types.proto:88]: Is optional in BICEPS, should also be optional here, i.e. be [UInt64Value](https://protobuf.dev/reference/protobuf/google.protobuf/#uint64-value)

Comment (jannast) [reply src/t2iapi/device/types.proto]: done

Comment (jannast) [reply src/t2iapi/device/types.proto]: done

File: src/t2iapi/device/types.proto [hunk 2]
```diff
@@ -79,3 +79,12 @@ enum CalibrationState{
     CALIBRATION_STATE_CALIBRATED = 3;
     CALIBRATION_STATE_OTHER = 4;
 }
+
+/*
+Represents a container for MDIB version attributes.
+ */
+message MdibVersion {
```
[the remark this answers is not in the document]
Comment (jannast) [reply src/t2iapi/device/types.proto]: done

File: src/t2iapi/biceps/mdibversiongroup.proto [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+option java_outer_classname = "MdibVersionGroupProto";
+
+import "google/protobuf/wrappers.proto";
+
+/*
+Represents the attributeGroup MdibVersionGroup (defined in IEEE Std 11073-10207-2017).
+*/
+message MdibVersionGroup {
+    google.protobuf.UInt64Value mdib_version = 1;    // optional MdibVersion when finishing the rpc
+    google.protobuf.StringValue sequence_id = 2;     // SequenceId when finishing the rpc
```
Comment (ldeichmann) [inline src/t2iapi/biceps/mdibversiongroup.proto:22]: SequenceId is not optional and should remain a string

Comment (jannast) [reply src/t2iapi/biceps/mdibversiongroup.proto]: done

**Review by ldeichmann — APPROVED** [review]

**Pull request merged**